chalicelib/checks/higlass_checks.py

```diff
@@ -1895,6 +1899,25 @@ def patch_file_higlass_uid(connection, **kwargs):
                 action_logs['registration_failure'][hit['accession']] = err_msg
                 continue
 
+            # Call the Flask server component on the Higlass server to copy beddb files
+            # from S3 to the local file system. Using mounted versions of these files is very slow. 
+            if payload['filetype'] == 'beddb':
+                copy_res = requests.post(
```

Comment:
No retry mechanism here? Might want to consider adding. Could be a pain to clean up if this does occasionally fail (not saying it will). If it does though it may be useful to have the ability to re-queue certain beddb files that fail, but if you've tested this and haven't seen it happen at all it may not be necessary.

Reply:
Thanks! I tested it and it seems to work fine so far. Whenever it failed, there was a larger issue (like permissions), so a retry wouldn't have helped anyway. 
Also, on initial registration, we can see if there was a problem, as it logs the copy failures. If it happens later and the file is not copied correctly, it will be automatically served from the S3 bucket.